### PR TITLE
Fix nullptr in PokePointer when you press any button

### PR DESCRIPTION
**Pull request by julenka**
Fixes: #4683

Also a couple additional fixes:
- Fixes a failing PlayMode button test by passing in updated prefab file name
- Remove some extra print statements from Interactable.

### CODE REVIEW

**Comment by julenka** [comment]
Don't complete this PR yet. I'm working on a playmode test that repros the P0 bug (press button with hand and then move hand off of button) that I'd like to check in as part of this PR.

File: Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/PokePointer.cs [hunk 1]
```diff
@@ -102,7 +102,7 @@ public override void OnPostSceneQuery()
                 return;
             }
 
-            if (Result?.CurrentPointerTarget != null)
+            if (Result?.CurrentPointerTarget != null && closestProximityTouchable != null)
```
Comment (julenka) [inline Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/PokePointer.cs]: @MenelvagorMilsom this is the actual fix :-) the other changes in this PR are to fix a playmode test, remove some extra logging, and then add a playmode test that repros this issue so we don't hit it again.

**Comment by julenka** [comment]
@thalbern and @MenelvagorMilsom I'm going to complete this PR because it fixes P0 issue, however keeping it open so we can discuss the tests and the fix. 

I'm not sure if my fix that I mention above (add a null check) has unexpected side effects, @MenelvagorMilsom would love for you to check that in case we need a follow up. I did test the hand interaction examples and touchables examples scenes and everything seemed fine.

I think there may be a better way to write the hand simulation for the tests, please don't consider this final. For example I was thinking to have a SimulatedHand class that would drive a simulated hand for the tests. It would look something like this:

```
class SimulatedHand
public SimulatedHand(Handedness h)
// Shows the hand in the open state
public IEnumerator Show()
// Hides the hand
public IEnumerator Hide()
public IEnumerator SetHandPose(gestureId)

// Couple methods below to move a hand over some number of frames
public IEnumerator MoveTo(fromPosition, toPosition, overHowManyFrames);
public IEnumerator MoveTo(toPosition, overHowManyFrames);
public IEnumerator SetPosition(toPoPosition);
```

**Comment by julenka** [comment]
x86 master build is passing locally for me. Perhaps this is an intermittent failure?

**Comment by julenka** [comment]
Update: I removed the PlayMode tests from this PR since they were breaking CI. will put them in a separate PR.

File: Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/PokePointer.cs (hunk 1 above)
Comment (MenelvagorMilsom) [reply Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/PokePointer.cs]: Makes sense to me! Thanks for catching this.

**Pull request merged**